docs/extensions/info-xml.md

```diff
@@ -360,20 +360,21 @@ Elements acceptable within `<urls>`
 ### `<ver>` {:#ver}
 
 * Containing element: [`<compatibility>`](#compatibility)
-* Description: a version of CiviCRM with which this extension is compatible; expressed as two digits
+* Description: a version of CiviCRM with which this extension is compatible; expressed as two digits, each `<ver>` element should only contain a single compatible version of CiviCRM.
 * Contains: text
 * Example: `4.7`
 
-!!! note "Point releases"
-    It is not currently possible to specify compatibility with point releases. If your extension is compatible with CiviCRM 4.7.21 but *not* 4.7.20, then you will need to clearly specify this in the [comments](#comments).
-
-!!! note "Forward compatibility (4.7/5.x)"
-    For CiviCRM 3.x and 4.x, `<ver>` tags must explicitly list all compatible versions.
-
-    For CiviCRM 4.7.x and 5.x, `<ver>` tags imply forward compatibility.
+!!! note "Which version to choose?"
+    Specify the minimum version number required by this extension. It is recommended to use 5.0 whenever possible. 
```

Comment:
@MikeyMJCO Not sure about *recommending* 5.0 here as a lot of stuff won't work with 5.0.0! My policy is to specify the last security release (so 5.24).

Reply:
That's a fair point.

Reply:
Removed that part.